src/tests/unit/tests/DevTools/target-page-inspector.test.ts

```diff
@@ -35,19 +50,48 @@ describe('TargetPageInspector', () => {
         'div:not(.awesome)',
         'div::after',
         '#result;button',
+        `body"); throw new Error("should not throw this error`,
+        `body'); throw new Error('should not throw this error`,
     ];
 
-    it.each(safeSelectors)('calls eval on the target page with selector = %s', safeSelector => {
-        const expectedScript =
-            'inspect(document.querySelector(' + JSON.stringify(safeSelector) + '))';
-
-        testSubject.inspectElement(safeSelector, testFrameUrl);
-
-        inspectedWindowMock.verify(
-            inspected => inspected.eval(expectedScript, { frameURL: testFrameUrl } as any),
-            Times.once(),
-        );
-    });
+    it.each(selectors)(
+        'calls eval through the inspected window, with safe selector = %s',
```

Comment:
I think the word "safe" in this string is a leftover?

Reply:
Updated.